pkg/actions/lua/lakefs/catalogexport/storage_utils.lua

```diff
@@ -0,0 +1,34 @@
+local utils = require("lakefs/catalogexport/internal")
+local pathlib = require("path")
+local url = require("net/url")
+
+--[[
+    ### Default exported table structure:
+
+    ${storageNamespace}
+    _lakefs/
+        exported/
+            ${ref}/
+                ${commitId}/
+                    ${tableName}/
+                        ...
+]]
+local function get_storage_uri_prefix(storage_ns, commit_id, action_info)
+    local branch_or_tag = utils.ref_from_branch_or_tag(action_info)
+    local sha = utils.short_digest(commit_id)
+    return pathlib.join("/", storage_ns, string.format("_lakefs/exported/%s/%s/", branch_or_tag, sha))
+end
+
+local function parse_storage_uri(uri)
```

Comment:
this function is implemented in `pkg/actions/lua/lakefs/catalogexport/internal.lua`

Not sure why we need another utils package?
the other called `internal.lua` to make sure customers don't use because otherwise we have to add documentation for it and support etc and servers as utils.